app/controllers/restaurants_controller.rb

```diff
@@ -14,7 +14,8 @@ def show
   # POST /restaurants
   def create
     # create! RecordInvalid exception handled in ApplicationController
-    Restaurant.create!(restaurant_params)
+    rest = Restaurant.create!(restaurant_params)
+    render_json(rest, :created)
   end
 
   # PUT /restaurants/:id
```

Comment:
do you know what is the difference between `@ restaurant ` and `restaurant` attribute?

Reply:
for my understanding @ is a way to declare member variable of the class. however I got contradicting explanations on how to use it especially with the ||= operator.
currently I just use it with before_action to trim repetitions of the code lines Restaurant.find(..)

Reply:
so let's talk about this :)